public/api/v1/expectedbuild.php

```diff
@@ -44,12 +44,7 @@
 $buildname = htmlspecialchars(pdo_real_escape_string($_REQUEST['name']));
 $buildtype = htmlspecialchars(pdo_real_escape_string($_REQUEST['type']));
 
-// Make sure the user has admin access to this project.
-if (!isset($_SESSION['cdash']) || !isset($_SESSION['cdash']['loginid'])) {
-    $response['requirelogin'] = 1;
-    echo json_encode($response);
-    return;
-}
+// Make sure the user has access to this project.
```

Comment:
Good point, renamed.  We still need this check for private projects, but we don't need to check for administrative rights on GET requests here.